docs-src/simple-gateway.md

```diff
@@ -0,0 +1,40 @@
+# Deploying a simple Gateway
+
+The simplest possible deployment is a Gateway and Route resources which are deployed together and have the same owner. This represents the same kind of model used for Ingress. A Gateway is deployed and an HTTPRoute is deployed which matches all traffic and sends it to a single Service named foo-svc. 
```

Comment:
A small nit - we usually try to wrap markdown files with 80 char line lengths where possible. There are a lot of tools that do this automatically for you, I use the [rewrap extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) for VSCode.